README.md

```diff
@@ -228,3 +232,8 @@ At any time a `demandA` or `demandB` can be commented on by any member by `POST`
 #### Rejecting match2
 
 At any time after a `match2` is `proposed`, and before it reaches `acceptedFinal` state, any of its members can reject the `match2` using [`POST /v1/match2/{id}/rejection`](http://localhost:8000/swagger/#/match2/RejectMatch2OnChain). Once a `match2` is rejected, it is permanently closed.
+
+#### Cancelling match2
+
+At any time after a `match2` is in `acceptedFinal` state, any of its members can cancel the `match2` using [`POST /v1/match2/{id}/cancellation`](http://localhost:3000/swagger/#/match2/CancelMatch2OnChain). Once a `match2` is rejected, it is permanently closed.
```

Comment:
Idk if it's worth saying that it is only MemberA or MemberB who can cancel, not optimiser (it's not like Reject, where even the optimiser could reject)